tools/run-dev.py

```diff
@@ -145,6 +145,7 @@
         manage_args + ['127.0.0.1:%d' % (tornado_port,)],
         ['./manage.py', 'process_queue', '--all'] + manage_args,
         ['env', 'PGHOST=127.0.0.1',  # Force password authentication using .pgpass
+         sys.executable,
```

Comment:
Let's add a comment above this and line-wrap this block to make it a bit more clear what problem this is solving.